front/src/components/ProtectedRoute.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { AppState } from 'store';
+import { Redirect } from 'react-router-dom';
+
+export interface ProtectedRouteProps {
```

Comment:
Could we remove the `export` if not needed, and rename this to just Props?

Reply:
sure